Meta/build-image-qemu.sh

```diff
@@ -122,7 +122,7 @@ if [ $USE_EXISTING -ne 1 ]; then
         if [ -x /sbin/mke2fs ]; then
```

Comment:
Not sure if we need the `if`-statement at all now.

Reply:
Yes, that can be removed. `/sbin` is covered by `find_executable`, and the actual arguments also don't seem to differ.